lib/registry.js

```diff
@@ -11,6 +15,23 @@ class Registry {
 		return 'application/openmetrics-text; version=1.0.0; charset=utf-8';
 	}
 
+	static get ESCAPE_STRING_REPLACE_MAP() {
```

Comment:
any particular reason to assign these as static props on the class?

Reply:
actually no, I prefer to have these evaluated only once per process, but wasn't sure about breaking the coding style.
what do you recommend? module level `const`?

Reply:
>wasn't sure about breaking the coding style.

Adding stuff as static props adds them to the public API (whether or not all of them _should_ is probably something we should revisit). In this case, I don't think there's any need to add more

> module level `const`?

yeah 👍